### ISSUE TITLE
Unable to select shorter named icons

### ISSUE THREAD

**Issue by jared-sheets** [mephitrpg/rpg-cards#38]
Expected behaviour: When attempting to use an icon located at game-icons.net, typing in the icon name into the 'Icon' field will generate a list of icons. Clicking on the icon name will place the icon onto the card.

Issue: Name of icon located at game-icons.net is not appearing in Icon list.

Steps to recreate:

I'm attempting to use the 'bo' icon located at http://game-icons.net/delapouite/originals/bo.html

Type in 'bo' into the 'Icon' field, and a list is generated of matching icon names.

Although several icons that begin with the letters 'bo' are found, no icon named 'bo' is displayed. If I simply type 'bo' and then tab to the next field, no icon is displayed. 

The issue is likely that the list generated for typing in 'bo' is too long to display all results and perhaps the 'bo' icon is displayed last on the list? I have attached a screenshot below for reference.

![bo](https://cloud.githubusercontent.com/assets/13834261/9306925/577a3b8a-44c9-11e5-8061-64c05748d5f2.PNG)

**Comment by crobi** [comment]
This, or the `bo` icon is not contained in the list at all - I update the list of icons fairly irregularly. As a workoround, simply type in `bo` and press enter.

**Comment by jared-sheets** [comment]
Understood -- perhaps the icon was added recently at game-icons. I looked up other images and found similar situations where icons are not listed in the drop down list, so I believe you are correct in that it is a new icon.

I did try to enter `bo` into the Icon field and press enter as a workaround, but it auto-completes to the top suggested icon, which is `boar-tusks`. I am going to close my issue, however, because I believe that stems back to the icon just not being in the list.
